keps/sig-storage/3476-volume-group-snapshot/README.md

```diff
@@ -830,21 +954,52 @@ details). For now, we leave it here.
 _This section must be completed when targeting beta graduation to a release._
 
 * **How does this feature react if the API server and/or etcd is unavailable?**
+  If the API server is unavailable, VolumeGroupSnapshot creation and deletion won't happen
+  until the API server is up again.
+
+  If the request is already being processed by the CSI driver and it is completed,
+  the CSI snapshotter sidecar won't be able to update VolumeGroupSnapshotContent
+  until the API server is up again. The CSI driver should support idempotency, so
+  a retry should not result in orphaned resources on the underlying storage system.
 
 * **What are other known failure modes?**
   For each of them, fill in the following information by copying the below template:
   - [Failure mode brief description]
```

Comment:
This part `[Failure mode brief description]` should be filled in here; the items below refer to how to detect that particular failure mode.